smithy-model/src/test/java/software/amazon/smithy/model/shapes/ModelSerializerTest.java

```diff
@@ -63,6 +63,14 @@ public void testRoundTrip(Path path) {
         ObjectNode expected = Node.parse(IoUtils.readUtf8File(path)).expectObjectNode();
 
         Node.assertEquals(actual, expected);
+
+        // Now validate the file is serialized correctly when downgraded to 1.0.
```

Comment:
Would we be able to have a test that starts a 1.0 model, loads it and serializes it as 1.0 and compares?